easybuild/easyblocks/n/numpy.py

```diff
@@ -244,8 +245,12 @@ def sanity_check_step(self, *args, **kwargs):
         }
         custom_commands = [
             ('python', '-c "import numpy"'),
-            ('python', '-c "import numpy.core._dotblas"'),  # _dotblas is required for decent performance of numpy.dot()
         ]
+        if LooseVersion(self.version) < LooseVersion("1.10"):
+            # _dotblas is required for decent performance of numpy.dot()
+            custom_commands.append (('python', '-c "import numpy.core._dotblas"'))
+        else:
+            custom_commands.append (('python', '-c "from numpy.core.multiarray import dot"'))
```

Comment:
turns out this is not a good check, the import always works, even if numpy was built non-optimized (no proper BLAS)

on a Haswell system, on top of `Python/2.7.10-intel-2015b`:

* deliberately broken numpy 1.10.1 installation (basically unpack + `python setup.py build --compiler=intel --fcompiler=intelem; python setup.py install --prefix ...`):

```
$ python -c "from numpy.core.multiarray import dot"; echo $?
0
$ python -m timeit -n 3 -r 3 "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"
3 loops, best of 3: 1.05 sec per loop
```

* proper numpy 1.10.1 install through EasyBuild

```
$ python -c "from numpy.core.multiarray import dot"; echo $?
0
$ python -m timeit -n 3 -r 3 "import numpy; x = numpy.random.random((1000, 1000))" "numpy.dot(x, x.T)"
3 loops, best of 3: 20.6 msec per loop
```

So, although the timed test we have will still catch the problem, it would be nice to figure out a better way to check.

And I guess we need to start considering significantly lowering the time limit, 1.05s is awfully close to 1000ms. :-)